src/infra/core/src/ingest_service_impl.rs

```diff
@@ -95,30 +736,23 @@ impl IngestServiceImpl {
             },
         }
     }
+}
 
-    async fn do_ingest(
+///////////////////////////////////////////////////////////////////////////////
+// Legacy Spark-based ingest path
+///////////////////////////////////////////////////////////////////////////////
+
+impl IngestServiceImpl {
```

Comment:
In a separate impl block I put all the legacy stuff that will be deleted once we drop Spark ingest.